src/main/java/com/google/codeu/servlets/StatsPageServlet.java

```diff
@@ -8,15 +11,29 @@
 import javax.servlet.http.HttpServletResponse;
 
 /**
- * Responds with a hard-coded message for testing purposes.
+ * A servlet for handling fetching site statistics.
  */
 @WebServlet("/stats")
 public class StatsPageServlet extends HttpServlet{
+
+  private Datastore datastore;
+
+  @Override
+  public void init() {
+    datastore = new Datastore();
+  }
   
- @Override
- public void doGet(HttpServletRequest request, HttpServletResponse response)
-   throws IOException {
-  
-  response.getOutputStream().println("hello world");
- }
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response)
```

Comment:
I would maybe add a comment explaining the format of the response? The description of this PR would be a perfect comment here.

Reply:
Comments added for both functions!